multicluster/service-mirror/cluster_watcher.go

```diff
@@ -192,18 +192,30 @@ func (rcsw *RemoteClusterServiceWatcher) originalResourceName(mirroredName strin
 	return strings.TrimSuffix(mirroredName, fmt.Sprintf("-%s", rcsw.link.TargetClusterName))
 }
 
-func (rcsw *RemoteClusterServiceWatcher) getMirroredServiceLabels() map[string]string {
-	return map[string]string{
-		consts.MirroredResourceLabel:  "true",
-		consts.RemoteClusterNameLabel: rcsw.link.TargetClusterName,
+func (rcsw *RemoteClusterServiceWatcher) getMirroredServiceLabels(remoteService *corev1.Service) map[string]string {
```

Comment:
Would be good to have a short comment for `getMirroredServiceLabels` to document the function usage, and more importantly, that its param `remoteService` is optional.